CONTRIBUTING.md

```diff
@@ -77,16 +77,18 @@ Code-level contributions to React Native generally come in the form of [pull req
 The process of proposing a change to React Native can be summarized as follows:
 
 1. Fork the React Native repository and create your branch from `master`.
-2. Make the desired changes to React Native sources. Use the `RNTester` app to test them out.
-3. If you've added code that should be tested, add tests.
-4. If you've changed APIs, update the documentation, which lives in [another repo](https://github.com/facebook/react-native-website/).
-5. Ensure the test suite passes, either locally or on CI once you opened a pull request.
-6. Make sure your code lints (for example via `yarn lint --fix`).
-7. Push the changes to your fork.
-8. Create a pull request to the React Native repository.
-9. Review and address comments on your pull request.
+2. Run `yarn` to install dependencies.
```

Comment:
If you number these all with `1.` then markdown will auto number them and future diffs will be simpler when adding steps